spec/spec.md

```diff
@@ -77,13 +76,13 @@ The following items are out of scope for the current version of this document:
     - Selective disclosure and unlinkability
     - Zero-Knowledge Proofs
     - Disputes
-- Non-native Self-Issued OPs like web applications, PWAs, etc.
+- Non-native Wallets like web applications, PWAs, etc.
 
-Note: Although selective disclosure and unlinkability are out of scope of this document, future versions will include JSON Web Proofs ([[ref:JWP]]) and JSON Web Algorithms ([[ref:JWA]]) once they get ratified in IETF.
+Note: Although selective disclosure and unlinkability are out of scope of this document, future versions will consider Selective Disclosure for JWTs ([[ref: SD-JWT]]), JSON Web Proofs ([[ref:JWP]]) and JSON Web Algorithms ([[ref:JWA]]) once they mature in IETF.
```

Comment:
we haven't discussed this yet so feels a bit soon to add? i guess the language is vague enough?